content/en/1.13.x/concepts/updating-bottlerocket.markdown

```diff
@@ -0,0 +1,299 @@
+3. Get the name of the ECS cluster where you are running Bottlerocket container instances.
+4. [Install the Bottlerocket ECS updater](https://github.com/bottlerocket-os/bottlerocket-ecs-updater#install) using a CloudFormation template (provided in the full ECS updater installation documentation, linked above).
+
+Detailed steps, including commands to run, are provided in the ECS updater documentation, linked above.
+
+**Note:** By default, the ECS updater checks for updates every 12 hours.
+
+### Kubernetes
+
+For the `aws-k8s-*` variants of Bottlerocket, there are a handful of possible ways to update Bottlerocket: Brupop, the EKS Console, `eksctl`, and SSM Command Documents.
+The following sections discuss each of these methods.
+
+#### Brupop
+
+This is the recommended method to update Bottlerocket on Kubernetes.
+
+##### Brupop: Release YAML
+
+How to install Brupop is covered in the [Brupop documentation](https://github.com/bottlerocket-os/bottlerocket-update-operator#installation).
+
+Essentially, you will need to do the following:
+
+1. Install `cert-manager`.
+2. Install `brupop`.
+3. Label your Bottlerocket nodes with the appropriate label.
+In most cases, you will want to use `bottlerocket.aws/updater-interface-version=2.0.0`.
+
+Detailed steps, including commands to run, are provided in the Brupop documentation, linked above.
+
+#### EKS
+
+When running the `aws-k8s-*` variants of Bottlerocket on EKS, you can use either the EKS Console or `eksctl` to update your Bottlerocket nodes if you do not want to use Brupop.
+
+##### EKS Console
+
+In order to update your Bottlerocket nodes in an EKS cluster using the EKS Console, you will need to do the following:
+
+1. Go to the EKS Console.
+For example, for `us-west-2`, it is: https://us-west-2.console.aws.amazon.com/eks/home?region=us-west-2#/clusters
+2. Click on the cluster name that you want to update.
+3. Go to the "Compute" tab.
+4. Under "Node groups", look for the node group that you want to update.
+5. In the "AMI Release Version" column, click on "Update now".
+6. A modal box will pop up.
+Choose your desired update strategy (e.g. "Rolling update") and press the "Update" button.
+
+This will update your Bottlerocket nodegroup, and subsequently your Bottlerocket nodes, to the latest version of Bottlerocket.
+
+Detailed instructions can be found in the [EKS User Guide, in the "AWS Management Console" tab](https://docs.aws.amazon.com/eks/latest/userguide/update-managed-node-group.html#mng-update).
+
+##### `eksctl`
+
+As described in the [EKS User Guide](https://docs.aws.amazon.com/eks/latest/userguide/update-managed-node-group.html#mng-update), you can also use `eksctl` to update your Bottlerocket nodes.
+For example, you can run the following command to update your Bottlerocket nodes to the latest version of Bottlerocket, replacing `NODEGROUP_NAME`, `CLUSTER_NAME`, and `REGION_CODE` with your own values:
+
+```bash
+eksctl upgrade nodegroup \
```

Comment:
I don't think we should be providing instructions on how to replace nodes with eksctl. Nothing about it is Bottlerocket-specific.

Reply:
I agree that node replacement is a cluster management topic, not Bottlerocket-specific. 

I'll consider whether this doc should be about "updating Bottlerocket clusters" or whether to stay focused on "updating Bottlerocket nodes".

Reply:
Agreed. @mchaker Can you reduce this down into an overview of what needs to happen then refer to docs for eksctl?

Reply:
Yes. I'll summarize what needs to happen, then refer to eksctl docs for specifics.